### ISSUE TITLE
Panilizer partially rotates footprint

### ISSUE THREAD

**Issue by tommy1394** [ThisIsNotRocketScience/GerberTools#147]
i have a design with a LFPAK33 footprint FET.  I am trying to panelize a 2x5 arraingment of a small PCB which uses this FET.  Gerber panelizer partially rotates the footprint for this LFPAK33 part.  Has anyone experienced this or have any suggestions

<img width="685" alt="Screenshot 2023-01-02 at 10 09 59 AM" src="https://user-images.githubusercontent.com/36419780/210266511-e1ef5ee1-a525-4c36-8ad5-799cfe999ef8.png">

**Comment by CRImier** [comment]
could you upload your gerber zip and .gerberset project file?

**Comment by tommy1394** [comment]
yes, let me put together a zip packager...do you want the saved panelizer configuration as well?

**Comment by CRImier** [comment]
yea, that's the `.gerberset`. I'm not the dev, but, this is what'd be useful for devs for triage

**Comment by tommy1394** [comment]
here are the files
[panelizer-rotate-issue.zip](https://github.com/ThisIsNotRocketScience/GerberTools/files/10333358/panelizer-rotate-issue.zip)

**Comment by tommy1394** [comment]
on a side note, this takes more than 8 hours to process... is this normal?

**Comment by StijnKuipers** [comment]
are you building from source or using a downloaded binary? What version are you on?

A lot of macro stuff has been fixed - but not in a released bin yet.

and yes please - upload the gerbers somewhere. 

8 hours sounds extremely odd
